learning_assistant/views.py

```diff
@@ -93,7 +93,12 @@ def post(self, request, course_id):
             }
         )
 
-        prompt_template = render_prompt_template(request, request.user.id, course_id, unit_id)
+        # the "course id" being passed in as a query parameter is really a course run id. For the sake of
+        # clarity here, the value is reassigned to a more aptly named variable.
+        course_run_id = course_id
```

Comment:
The current variable name is a little confusing. The query parameter should really be `course_run_id`, but that would require a change to the URL name and an update to the frontend to account for that. 

This change that I've added here should be temporary, as I needed to move the call to `get_course_id` into the view, as both `render_prompt_template` and `get_chat_response` need the course id.